### DISCUSSION TITLE
Adding image into excel

### DISCUSSION THREAD

**Discussion by christianangon** [exceljs/exceljs#2015] [Q&A] [answered]
How do you simply add a url into the excel sheet? Im using Vuejs2 
and I use the logo.png as a sample for image located at assets
assets/logo.png

app.vue
const imageId1 = workbook.addImage({
      filename: '../assets/logo.png',
      extension: 'png',
    });

**Comment by christianangon** [comment]
This is the error message i received 
TypeError: u.readFile is not a function

**Comment by skypesky** [comment, marked as answer]
@spellman1980 
Maybe this can help you !
https://github.com/exceljs/exceljs/issues/1216#issuecomment-1047625753

**Reply by christianangon** [reply]
how do you add but from locally?

**Reply by skypesky** [reply]
@spellman1980  The method is the same.Use **absolute path** or **relative path**.
ex: 
```typescript
const response = await fetch('./test.jpg');
```
